demisto_sdk/commands/validate/validators/RN_validators/RN114_validate_release_notes_header.py

```diff
@@ -0,0 +1,255 @@
+    CONTENT_TYPE_SECTION_REGEX,
+    RN_HEADER_BY_FILE_TYPE,
+)
+from demisto_sdk.commands.content_graph.common import (
+    ContentType,
+)
+from demisto_sdk.commands.content_graph.objects.content_item import ContentItem
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Pack
+RN_HEADER_BY_CONTENT_TYPE = {
+    ContentType.PLAYBOOK: "Playbooks",
+    ContentType.INTEGRATION: "Integrations",
+    ContentType.SCRIPT: "Scripts",
+    ContentType.INCIDENT_FIELD: "Incident Fields",
+    ContentType.INDICATOR_FIELD: "Indicator Fields",
+    ContentType.INCIDENT_TYPE: "Incident Types",
+    ContentType.CLASSIFIER: "Classifiers",
+    ContentType.LAYOUT: "Layouts",
+    ContentType.REPORT: "Reports",
+    ContentType.WIDGET: "Widgets",
+    ContentType.DASHBOARD: "Dashboards",
+    ContentType.CONNECTION: "Connections",
+    ContentType.MAPPER: "Mappers",
+    ContentType.PREPROCESS_RULE: "PreProcess Rules",
+    ContentType.GENERIC_DEFINITION: "Objects",
+    ContentType.GENERIC_MODULE: "Modules",
+    ContentType.GENERIC_TYPE: "Object Types",
+    ContentType.GENERIC_FIELD: "Object Fields",
+    ContentType.LIST: "Lists",
+    ContentType.JOB: "Jobs",
+    ContentType.PARSING_RULE: "Parsing Rules",
+    ContentType.MODELING_RULE: "Modeling Rules",
+    ContentType.CORRELATION_RULE: "Correlation Rules",
+    ContentType.XSIAM_DASHBOARD: "XSIAM Dashboards",
+    ContentType.XSIAM_REPORT: "XSIAM Reports",
+    ContentType.TRIGGER: "Triggers Recommendations",  # https://github.com/demisto/etc/issues/48153#issuecomment-1111988526
+    ContentType.WIZARD: "Wizards",
+    ContentType.XDRC_TEMPLATE: "XDRC Templates",
+    ContentType.LAYOUT_RULE: "Layout Rules",
+    ContentType.ASSETS_MODELING_RULE: "Assets Modeling Rules",
+    ContentType.CASE_LAYOUT_RULE: "Case Layout Rules",
+    ContentType.CASE_FIELD: "Case Fields",
+    ContentType.CASE_LAYOUT: "Case Layouts",
+}
+CONTENT_TYPE_BY_RN_HEADER = {
+    header: content_type for content_type, header in RN_HEADER_BY_CONTENT_TYPE.items()
+}
+
+
+class ReleaseNoteHeaderValidator(BaseValidator[ContentTypes]):
+    error_code = "RN114"
+    description = (
+        "Validate the 2nd headers (the content items) are exists in the pack and having the right display"
+        " name."
+    )
+    rationale = (
+        "Provide documentation with clear headers for all modifications to make content usage easier."
+        " Validate headers for accuracy."
+    )
+    error_message = (
+        "The following invalid headers were found in:\n"
+        "{content_type_message}\n{content_item_message}\n"
+        "For common troubleshooting steps, please review the documentation found here: "
+        "https://xsoar.pan.dev/docs/integrations/changelog#common-troubleshooting-tips"
```

Comment:
I don't think that in this particular validation, there's much value sending users to that page, better remove the link imo.